### PR TITLE
docs: instructions for initial user creation

### PR DESCRIPTION
**Pull request by privateseabass**
#### Before submitting th PR
- [x] Read the [CONTRIBUTING document](https://github.com/monicahq/monica/blob/master/CONTRIBUTING.md) before submitting your PR.
- [x] If the PR is related to an issue or fix one, don't forget to indicate it.
- [ ] Create your PR as draft if it is not final yet. Mark it as ready... when it’s ready. Otherwise the PR will be considered complete and rejected if it's not working.

### General checks
- [x] Make sure that the change you propose is the smallest possible.
- [ ] The name of the PR should follow the [conventional commits guideline](https://github.com/monicahq/monica/blob/master/docs/contribute/index.md#conventional-commits) that the project follows.

### Front-end changes
- [ ] If you change the UI, make sure to ask repositories administrators first about your changes by pinging djaiss or asbiin in this PR.
- [ ] Screenshots are included if the PR changes the UI.
- [ ] Front-end tests have been written with Cypress.

#### Backend/models changes
- [ ] The API has been updated.
- [ ] API's documentation has been added by submitting a pull request in the [marketing website repository](https://github.com/monicahq/marketing_site/pulls).
- [ ] Tests have been added for the new code.
- [ ] If you change a model, make sure the SetupTest file is updated. We need seeders to develop locally and generate fake data.

#### If the code changes the SQL schema
- [ ] Make sure exporting account data as SQL is still working.
- [ ] Make sure your changes do not break importing data with `vCard` and `.csv` files.
- [ ] Make sure account reset and deletion still work.

#### Other tasks
- [ ] [CHANGELOG](https://github.com/monicahq/monica/blob/master/CHANGELOG.md) entry added, if necessary, under `UNRELEASED`.
- [ ] [CONTRIBUTORS](https://github.com/monicahq/monica/blob/master/CONTRIBUTORS) entry added, if necessary.
- [ ] If it's relevant and worth mentioning, create a changelog entry for this change. The changelog entry will appear inside the UI for all users to see. To know if your change is worth the creation of a changelog entry, [read the documentation](https://github.com/monicahq/monica/blob/master/docs/administrators/tips.md#when-is-it-relevant-to-create-a-changelog-entry).
- [ ] Don't forget to [ask for a free account](mailto:regis@monicahq.com) on https://monicahq.com as anyone who contributes can request a free account.

#3764 only fixed for general install.
#3762
#3744

### CODE REVIEW

**Comment by privateseabass** [comment]
#3764 only fixed for general install.
#3762
#3744

Kind of new to the contributions, let me know if I missed anything in that list

**Comment by asbiin** [comment]
Thank you for your help @privateseabass 
Your proposal is interesting, but I don't think we should write it like this.
- `php artisan setup:production` works for everyone. This setup a new instance, and let the user open its browser to create the first user. 
- `php artisan setup:production --email=your@email.com --password=yourpassword -v` should only be used if you absolutely want to create the first user by command.

**Comment by privateseabass** [comment]
@asbiin This hasn't been my experience.
I open the browser and it has consistently given me a login screen.
What is the recommended method of creating the first user?
Or what in the setup am I missing that makes the first user?

**Comment by asbiin** [comment]
@privateseabass 

> I open the browser and it has consistently given me a login screen.

When there is no user registered (clear install), it redirects you to the register page.
In any case, you can go to the register page to create a new user ...

**Comment by privateseabass** [comment]
What is the link to the register page?
There is no button or link for it on my installs for Debian and Ubuntu.

Would that mean that the [Debian install instructions](https://github.com/monicahq/monica/blob/master/docs/installation/debian.md) and [Ubuntu install instructions](https://github.com/monicahq/monica/blob/master/docs/installation/ubuntu.md) are not clear installs?

I followed only those instructions, after all.

Which instructions are recommended instructions for installation?

**Comment by fklappan** [comment]
> 
> 
> @privateseabass
> 
> > I open the browser and it has consistently given me a login screen.
> 
> When there is no user registered (clear install), it redirects you to the register page.
> In any case, you can go to the register page to create a new user ...

See #3762
I for sure had a clean install (just installed a new home server) but had already two users in the database: John Doe (admin@admin.com) and Blank State (blank@blank.com).

Only "error" I made during setup was using PHP 7.2 instead of 7.3. Used Tag 2.17.0 from master.

File: docs/installation/debian.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ exit
 1. Update `.env` to your specific needs. Don't forget to set `DB_USERNAME` and `DB_PASSWORD` with the settings used behind. You'll need to configure a [mailserver](/docs/installation/mail.md) for registration & reminders to work correctly.
 1. Run `composer install --no-interaction --no-suggest --no-dev --ignore-platform-reqs` to install all packages.
 1. Run `php artisan key:generate` to generate an application key. This will set `APP_KEY` with the right value automatically.
-1. Run `php artisan setup:production -v` to run the migrations, seed the database and symlink folders.
+1. Run `php artisan setup:production --email=your@email.com --password=yourpassword -v` to run the migrations, seed the database, symlink folders and setup the first user account.
```
Comment (asbiin) [inline docs/installation/debian.md:114]: I propose another solution:
```suggestion
1. Run `php artisan setup:production -v` to run the migrations, seed the database and symlink folders.
    - You can use `email` and `password` parameter to setup a first account directly: `php artisan setup:production --email=your@email.com --password=yourpassword -v`
```

File: docs/installation/ubuntu.md [hunk 2]
```diff
@@ -116,7 +116,7 @@ exit
 1. Update `.env` to your specific needs. Don't forget to set `DB_USERNAME` and `DB_PASSWORD` with the settings used behind. You'll need to configure a [mailserver](/docs/installation/mail.md) for registration & reminders to work correctly.
 1. Run `composer install --no-interaction --no-suggest --no-dev --ignore-platform-reqs` to install all packages.
 1. Run `php artisan key:generate` to generate an application key. This will set `APP_KEY` with the right value automatically.
-1. Run `php artisan setup:production -v` to run the migrations, seed the database and symlink folders.
+1. Run `php artisan setup:production --email=your@email.com --password=yourpassword -v` to run the migrations, seed the database, symlink folders and setup the first user account.
```
Comment (asbiin) [inline docs/installation/ubuntu.md:119]:
```suggestion
1. Run `php artisan setup:production -v` to run the migrations, seed the database and symlink folders.
    - You can use `email` and `password` parameter to setup a first account directly: `php artisan setup:production --email=your@email.com --password=yourpassword -v`
```

**Pull request merged**